slosilo.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |gem|
   gem.name          = "slosilo"
   gem.require_paths = ["lib"]
   gem.version       = Slosilo::VERSION
-  gem.required_ruby_version = '>= 1.9.3'
+  gem.required_ruby_version = '>= 3.0.2'
```

Comment:
What do you think about changing the required ruby version to simply be `>= 3.0.0`?  That'll ensure any version with the Ruby 3 `to_proc` change is handled.